velox/connectors/hive/storage_adapters/s3fs/S3Util.h

```diff
@@ -57,8 +77,15 @@ inline std::string s3URI(const std::string& bucket, const std::string& key) {
 }
 
 inline std::string s3Path(const std::string_view& path) {
-  // Remove the prefix S3:// from the given path
-  return std::string(path.substr(kS3Scheme.length()));
+  // Remove the prefix 's3://' or one off 'oss://' 's3a://' from the given path
```

Comment:
nit: full stop at the end.

Reply:
Could simplify write as `Remove one of the prefixes 's3://', 'oss://', 's3a://' if any from the given path.`

Reply:
fixed